fhireval/data/BasicTest.json

```diff
@@ -0,0 +1,328 @@
+{
+	"groupname": "BasicTest",
+	"datatype": "TestOnly",
+	"resources": {
+		"Patient" : [
+			{
+				"resourceType": "Patient",
+				"extension": [
+				  {
+					"url": "http://hl7.org/fhir/us/core/StructureDefinition/us-core-race",
+					"extension": [
+					  {
+						"url": "ombCategory",
+						"valueCoding": {
+						  "system": "urn:oid:2.16.840.1.113883.6.238",
+						  "code": "2028-9",
+						  "display": "Asian"
+						}
+					  },
+					  {
+						"url": "text",
+						"valueString": "Asian:Middle Eastern (from the United Arab Emirates)"
+					  }
+					]
+				  }, {
+					"url": "https://ncpi-fhir.github.io/StructureDefinition/visit-number",
+					"valueInteger": 1
+				  }
+				],
+				"identifier": [
+				  {
+					"system": "urn:ncpi-test:unique-string",
+					"value": "TEST-Patient-1"
+				  }
+				],
+				"meta": {
+				  "profile": [
+					"http://hl7.org/fhir/StructureDefinition/Patient"
+				  ]
+				}
+			  }
+		],
+		"Observation": [
+			{
+				"resourceType": "Observation",
+				"extension": [
+				  {
+					"url": "http://ncpi-fhir.github.io/StructureDefinition/visit-number",
+					"valueInteger": 1
+				  }
+				],
+				"identifier": [
+				  {
+					"system": "urn:ncpi-test:unique-string",
+					"value": "BasicTest-Observation-1"
+				  }
+				],
+				"status": "final",
+				"code": {
+				  "coding": [
+					{
+					  "system": "http://loinc.org",
+					  "code": "8302-2",
+					  "display": "Body height"
+					},
+					{
+					  "system": "https://www.ohdsi.org",
+					  "code": "3036277",
+					  "display": "Body height"
+					}
+				  ],
+				  "text": "Body height"
+				},
+				"subject": {
+				  "reference": "Patient/TEST-Patient-1"
+				},
+				"_effectiveDateTime": {
+				  "extension": [
+					{
+					  "url": "http://hl7.org/fhir/StructureDefinition/cqf-relativeDateTime",
+					  "extension": [
+						{
+						  "url": "target",
+						  "valueReference": {
+							"reference": "Patient/TEST-Patient-1"
+						  }
+						},
+						{
+						  "url": "targetPath",
+						  "valueString": "birthDate"
+						},
+						{
+						  "url": "relationship",
+						  "valueCode": "after"
+						},
+						{
+						  "url": "offset",
+						  "valueDuration": {
+							"value": 1.911,
+							"unit": "a",
+							"system": "http://unitsofmeasure.org",
+							"code": "years"
+						  }
+						}
+					  ]
+					}
+				  ]
+				},
+				"valueQuantity": {
+				  "value": 165.74710439786688,
+				  "unit": "cm",
```

Comment:
`"centimeter"`

Reply:
After some further consideration, we've decided to leave these as is and will likely discuss issues like this with the larger group at some point.